### PR TITLE
[#796] Rename ex.Scene.children to ex.Scene.actors

### LINKED ISSUE #796: Rename ex.Scene.children to ex.Scene.actors
### Context
Currently, it is conceptually confusing when trying to get at actors in a scene. It doesn't make sense that the collection in a scene is called children. It should be named for what it is :) 

This will be a breaking change, so we could potentially add backwards compatibility getter/setter to call through to the new `ex.Scene.actors`. Something like the below code perhaps with a deprecation warning decorator? (unfortunately decorators in TS can't be added to a getter AND it's setter 😢 )

image

```typescript
/**
  * The actors in the current scene
  */
 public actors: Actor[] = [];

 @obsolete({ alternateMethod: 'ex.Scene.actors' })
 public set children (actors: Actor[]) {
    this.actors = actors;
 }

 public get children (): Actor[] {
    return this.actors;
 }
```

### Proposal

* Refactor `ex.Scene.children` to be renamed to `ex.Scene.actors`
* Update specs to reflect change
* Update visual tests in /sandbox/tests

### PR DESCRIPTION
**Pull request by Crizzooo**
Closes #796

## Changes:

- changed `ex.Scene.children` to be `ex.Scene.actors`
- updated test specs to reflect this in SceneSpec.ts
- did not make changes in Sandbox and was unsure of what is needed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ at anywhere from mute (`volume` is 0.0) to full volume (`volume` is 1.0). ([#801
 - Added another DisplayMode option: `DisplayMode.Position`. When this is selected as the displayMode type, the user must specify a new `position` option
 - Added a new Interface `AbsolutePosition` which can described the `position` option. A `string` can also describe `position`
 - Added a static method distanceBetweenVectors to the Vector class (#517)[https://github.com/excaliburjs/Excalibur/issues/517]
+- Changed `ex.Scene.children` to be `ex.Scene.actors`, and updated unit test specs to reflect this change (#796)[https://github.com/excaliburjs/Excalibur/issues/796#issuecomment-304422629]
```
Comment (kamranayub) [inline CHANGELOG.md]: Remove hash (#) anchor reference

**Comment by kamranayub** [comment]
Just need to fix TSLint warning and you're good

**Comment by Crizzooo** [comment]
Hm, yeah I was getting errors when I tried to `npm all`. What do I need to do to get this PR merged?

File: src/engine/Scene.ts [hunk 2]
```diff
@@ -574,4 +574,13 @@ export class Scene extends Class {
          }
       }
    }
-}
\ No newline at end of file
+
+   @obsolete({ alternateMethod: 'ex.Scene.actors' })
```
Comment (kamranayub) [inline src/engine/Scene.ts]: Need to import `Obsolete` decorator from decorators folder

**Comment by kamranayub** [comment]
@Crizzooo not sure if you got a comment notification, but it looks like all you need to do is import the obsolete decorator to get this working and we can merge it!

**Comment by Crizzooo** [comment]
Thanks for the reminder, @kamranayub !

**Comment by eonarheim** [comment]
@Crizzooo Thanks for helping!

**Pull request merged**